### DISCUSSION TITLE
Can you connect to code-server from an existing vs code install?

### DISCUSSION THREAD

**Discussion by bgv2** [coder/code-server#2557] [Q&A] [answered]
What I mean is using a VS code install as a frontend to code-server. This is already possible using GitHub Codespaces.

**Comment by code-asher** [comment, marked as answer]
Unfortunately VS Code's remote extensions only support connecting to
other VS Code instances like Codespaces.

We have an issue for re-implementing these extensions so they can
connect to code-server: #1315

**Reply by bgv2** [reply]
ok, thanks
